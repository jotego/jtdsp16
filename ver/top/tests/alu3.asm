x=0x20
y=0x30
a0=a0+p,p=x*y  # p=0x600
a0=a0+p  # a0 = 0x600
y=0
yl=0x40f # y = 0x40f
a1=a0&y  # a1 = 0x400
a0=a0^y  # a1 = 0x20f
end:
goto end
