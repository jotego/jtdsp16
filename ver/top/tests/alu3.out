r0=0x0000
r1=0x0000
r2=0x0000
r3=0x0000
rb=0x0000
re=0x0000
j =0x0000
k =0x0000
pr=0x0000
pt=0x0000
i =0x0000
a0=0x00000020f
a1=0x000000400
x=0x0020
y=0x0000040f
p=0x00000600
c0=0x0000
c1=0x0000
c2=0x0000
auc=0x0000
psw=0x0000
